package.json

```diff
@@ -75,7 +74,7 @@
     "suppress-exit-code": "1.0.0",
     "tmp-promise": "3.0.3",
     "tree-kill": "1.2.2",
-    "ts-node": "10.5.0",
+    "ts-node": "10.8.0",
```

Comment:
No particular reason for upgrading this package, just thought it could help avoid some bugs.